core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.spec.ts

```diff
@@ -84,6 +85,7 @@ describe('Translation Suggestion Review Modal Controller', function() {
         $scope: $scope,
         $uibModalInstance: $uibModalInstance,
         initialSuggestionId: 'suggestion_1',
+        subheading: subheading,
         reviewable: reviewable,
         suggestionIdToSuggestion: angular.copy(suggestionIdToSuggestion)
       });
```

Comment:
Shouldn't the below test be modified (to test the subheading)?

Reply:
done